x/staking/types/commission.go

```diff
@@ -10,24 +10,22 @@ import (
 type (
 	// Commission defines a commission parameters for a given validator.
 	Commission struct {
-		Rate          sdk.Dec   `json:"rate"`            // the commission rate charged to delegators, as a fraction
-		MaxRate       sdk.Dec   `json:"max_rate"`        // maximum commission rate which this validator can ever charge, as a fraction
-		MaxChangeRate sdk.Dec   `json:"max_change_rate"` // maximum daily increase of the validator commission, as a fraction
-		UpdateTime    time.Time `json:"update_time"`     // the last time the commission rate was changed
+		CommissionRates `json:"commission_rates"`
```

Comment:
Hmmm, so this would actually make it API breaking. Why not just simply embed the `CommissionRates` w/o a JSON tag? This way it won't be API breaking.

Reply:
I suppose I should move the pending log to "improvements" or leave under "breaking"?

Reply:
Have it under improvements :)